### DISCUSSION TITLE
Option to sell raid banners

### DISCUSSION THREAD

**Discussion by SkeletonAce** [Zrips/CMI#6266] [Enhancement]
I'm having difficulty adding raid banners to the sell list without enabling renamed tools etc to also be sold.
I want to do this is because my sellwand plugin hooks into cmi for worth values and so chests from raid farms and vindicator spawners cannot be sold entirely leaving the raid banners behind. 

Possibly adding an option to sell worthless items for $0 could also fix this

Probably not possible but it would be nice

**Comment by mrfloris** [comment]
It would be helpful for the overview if the title of a suggestion makes sense and doesn't say 'suggestions'. If you can still edit it, please do so.

It's a regular white banner with a design, I think it's out of scope for CMI to support every possible banner design. Like the ones in the end structures, pillager outposts, etc. 

<img width="390" alt="Screenshot 2022-08-20 at 13 11 50" src="https://user-images.githubusercontent.com/28841349/185743301-fd696cd1-604f-44cc-9129-a3a793b24893.png">
